video/out/hwdec/hwdec_vaapi.c

```diff
@@ -518,8 +542,12 @@ static void determine_working_formats(struct ra_hwdec *hw)
     p->probing_formats = false;
 
     MP_DBG(hw, "Supported formats:\n");
-    for (int n = 0; p->formats && p->formats[n]; n++)
-        MP_DBG(hw, " %s\n", mp_imgfmt_to_name(p->formats[n]));
+    for (int n = 0; p->formats && p->formats[n]; n++) {
+        MP_DBG(hw, "%s, drm format '%s(%016lx)'\n",
+                mp_imgfmt_to_name(p->formats[n]),
+                mp_tag_str(p->supported_wl_formats[n].format),
+                p->supported_wl_formats[n].modifier);
```

Comment:
1) there is no guarantee that supported_wl_formats is as long as formats
2) most people do not use wldmabuf so this will just print useless and potentially incorrect data

Reply:
Good point - fixed.